bin/civihr-install

```diff
@@ -596,8 +609,7 @@ export PATH="$PATH:$BINDIR"
 civihr_parse "$@"
 
 # Checking for Drush dependencies if not present downloading into temp.
-command -v drush >/dev/null 2>&1 || ( echo "[[Downloading dependencies.........]]" && curl -Lo "$TMPDIR/drush" "https://github.com/drush-ops/drush/releases/download/8.1.8/drush.phar"  && chmod +x "$TMPDIR/drush" && export PATH="$PATH:$TMPDIR/drush" )
-
+command -v drush >/dev/null 2>&1 || ( echo "[[Downloading dependencies.........]]" && curl -Lo "$TMPDIR/drush" "https://github.com/drush-ops/drush/releases/download/8.1.8/drush.phar" && chmod +x "$TMPDIR/drush" && export PATH="$TMPDIR:$PATH" )
```

Comment:
I'm not sure whether its a bad practice to modify temporary PATH variable within script and also I've seen Buildkit modifying PATH variable so thought that we can use same process.
And I've investigated on the 3 option you suggested in that case I think we need to then replace all the drush commands with the absolute variable substitution. Something like
```
drush -y make --concurrency=5 --working-copy "$MAKEFILE" "$WEB_ROOT"
```
would be replaced with 
```
$DRUSH -y make --concurrency=5 --working-copy "$MAKEFILE" "$WEB_ROOT"
```
Please correct if I'm wrong as I'm very new to shell scripting. thanks 
@aljazceru @AkA84 what you think about above suggestions by Davi ?